mlrun/runtimes/nuclio/api_gateway.py

```diff
@@ -389,10 +392,10 @@ def invoke(
         ):
             # inject access key from env
             auth = NuclioAuthInfo().from_envvar().to_requests_auth()
```

Comment:
What if user doesn't have this envvar set? I think it would improve UX if we warn a user about it.

Reply:
👍 will raise exception in that case